build.gradle.kts

```diff
@@ -149,11 +149,28 @@ allprojects {
         }
     }
 
-
-
     tasks.withType<Test> {
-        useJUnitPlatform()
+        // Target all type of test e.g. -DRunAllTests="true"
+        val runAllTests: String = System.getProperty("RunAllTests", "false");
```

Comment:
All the properties used in the EDC are in camel case with the first letter in lowercase, I would maintain this convention, so this would be called `runAllTests`

Reply:
Updated. thanks